test/operatorsource/helpers.go

```diff
@@ -0,0 +1,127 @@
+package operatorsource
```

Comment:
Do we really need these helper functions? I don't really see the add value in these wrapper methods vs just making the call directly? Especially since it does not do any sort of additional processing here.

This could also easily get out of hand when getting other resources. 

It also seems somewhat inconsistent. In other places we just make the call directly.

Reply:
I feel it would be a good idea to isolate the helpers method and even though this are wrappers methods this would make for clean and readable test. And also in future test cases this helpers methods can be reused so reducing the number of lines of code. For this test it might look like extra work but they could come in handy for adding new test cases in operatorsource.

Reply:
GetSubscription(), GetInstallPlan() etc... probably are more of a setback then help since nothing is done with the error condition. Seems like extra overhead with no add value.

We haven't done this in our other tests and i guess i'm having a hard time understanding what benefits it really adds. Perhaps the team has a different opinion? @baijum @akashshinde @Avni-Sharma @DhritiShikhar

Reply:
Creating a function for code readability is overkill. It also makes error traceback convoluted